hi-ml-azure/src/health_azure/himl.py

```diff
@@ -493,14 +493,14 @@ def submit_run_v2(workspace: Optional[Workspace],
     :param docker_shm_size: The Docker shared memory size that should be used when creating a new Docker image.
     :param wait_for_completion: If False (the default) return after the run is submitted to AzureML, otherwise wait for
         the completion of this run (if True).
-    :param wait_for_completion_show_output: If wait_for_completion is True this parameter indicates whether to show the
-        run output on sys.stdout.
     :param workspace_config_path: If not provided with an AzureML Workspace, then load one given the information in this
         config
     :param ml_client: An Azure MLClient object for interacting with Azure resources.
     :param hyperparam_args: A dictionary of hyperparameter search args to pass into a sweep job.
     :param num_nodes: The number of nodes to use for the job in AzureML.
-    :param processes_per_node: The number of processes per node to use for the job in AzureML.
+    :param pytorch_processes_per_node: For plain PyTorch multi-GPU processing: The number of processes per node. This
```

Comment:
This function shouldn't be called at all in a v1 job so this statement about being ignored in v1 is perhaps confusing